### ISSUE TITLE
Install gouml with error

### ISSUE THREAD

**Issue by david9991** [kazukousen/gouml#49]
I am trying to install gouml, but get this error.

```
go get -u github.com/kazukousen/gouml/cmd/gouml
# github.com/kazukousen/gouml/cmd/gouml
../../../github.com/kazukousen/gouml/cmd/gouml/main.go:17:22: cannot use cli.StringSliceFlag literal (type cli.StringSliceFlag) as type cli.Flag in array or slice literal:
	cli.StringSliceFlag does not implement cli.Flag (Apply method has pointer receiver)
../../../github.com/kazukousen/gouml/cmd/gouml/main.go:21:22: cannot use cli.StringSliceFlag literal (type cli.StringSliceFlag) as type cli.Flag in array or slice literal:
	cli.StringSliceFlag does not implement cli.Flag (Apply method has pointer receiver)
../../../github.com/kazukousen/gouml/cmd/gouml/main.go:25:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
	cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
../../../github.com/kazukousen/gouml/cmd/gouml/main.go:33:15: cannot use []cli.Command literal (type []cli.Command) as type []*cli.Command in assignment
../../../github.com/kazukousen/gouml/cmd/gouml/main.go:58:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
```

**Comment by nelsw** [comment]
Had the same issue, created pull request: https://github.com/kazukousen/gouml/pull/50

**Comment by kazukousen** [comment]
Thanks @david9991 and @nelsw

**Comment by kazukousen** [comment]
sorry, i fixed #51

**Comment by nelsw** [comment]
Maybe forgot to pull master before committing lol

I’d be happy to contribute regularly, fix more of your issues, if you’d considering mentioning me as a contributor or reference me in the readme.md :)
